src/screens/WalletScreen.js

```diff
@@ -152,6 +158,21 @@ export default function WalletScreen() {
     }
   }, [assetsSocket, dispatch, fetchedCharts, initialized, sections]);
 
+  useEffect(() => {
+    if (initialized) {
+      InteractionManager.runAfterInteractions(() => {
```

Comment:
Just realized that we `setInitialized` right after we call `initializeWallet` (which is an async function which we are not awaiting before we do `setInitialized`) - cannot recall if this is on purpose or if there are some assumptions of what has run in `initializeWallet` we are relying on before we use `initialized`

Reply:
Thinking out loud, need to still think more:

So assuming we do want to actually wait until we have fully initialized before calling `setInitialized`:
Since `initializeWallet` not only loads account data but triggers requests to fetch fresh data, there is a possible race condition where these 3 loads (global late, discover, profile) could overwrite fresh data triggered from `initializeWallet`
We may want to consider splitting out `initializeWallet` a bit further.

Reply:
Here's a breakdown:
`useLoadGlobalLateData`
* contacts (needed by fresh txns)
* wallet balances & names loaded to query cache (this is fine)
* nonce manager (this is fine)

`initializeDiscoverData`
* user lists (this is fine as long as it is after favorites)
* top movers (this could get refreshed data)

`initializeProfileData`
* txn signatures (needed by fresh txns)